hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1459,54 +1459,51 @@
 
     // SRAM (pre-verified IP) integration tests:
     {
-      name: chip_sram_access
-      desc: '''Verify access to the SRAM.
+      name: chip_sram_scrambled_access
+      desc: '''Verify scrambled memory accesses to both main and retention SRAMs.
 
-            Verify that the CPU can fetch data from the SRAM. Nothing extra needs to be done
-            here - all SW tests do this anyway.
-            '''
-      milestone: V2
-      tests: []
-    }
-    {
-      name: chip_sram_scramble
-      desc: '''Verify scrambling of the SRAM data.
+            - Trigger both SRAMs to fetch a new key and nonce from the OTP_CTRL
+            - Drive the CPU to perform random accesses to both RAMs and verify these operations
+              complete successfully by using the backdoor interface
 
-            SW enables scrambling within the SRAM. Ensure that the data written to the SRAM is
-            scrambled and likewise, when read from the SRAM is de-scrambled correctly via backdoor.
-            The key and nonce for the scrambling is supplied by the OTP.
+            Not much work needs to be done here, SW tests access RAM anyways.
```

Comment:
Nit: this seems to be an unnecessary detail - remove it.

Reply:
sure, thanks for the comment!